src/main/java/rx/internal/operators/OperatorMulticast.java

```diff
@@ -38,14 +38,16 @@
  *            the result value type
  */
 public final class OperatorMulticast<T, R> extends ConnectableObservable<R> {
-    final Observable<? extends T> source;
-    final Object guard;
-    final Func0<? extends Subject<? super T, ? extends R>> subjectFactory;
+    private final Observable<? extends T> source;
```

Comment:
I try to avoid private because the compiler generates accessor methods which slows down the interpreter phase and makes an extra hop while debugging methods.